index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Portfolio</title>
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css">
+	<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+	<link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+	<div class="app-container">
+		<nav class="app-nav">
+			<a id="logo" href="#">My logo</a>
+			<i id="menu-icon" class="bi bi-list"></i>
+		</nav>
+		<main class="app-greeting">
+			<h1>I'm Sane great to see you!</h1>
+			<p>I'm a software developer! I can help you build a product, feature or website. Look through some of my work
+				and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+			<p id="lets-connect">LET'S CONNECT</p>
+			<div class="socials">
+				<a href="#"><i class="bi bi-twitter"></i></a>
+				<a href="#"><i class="bi bi-linkedin"></i></a>
+				<a href="#"></a><i class="bi bi-github"></i></a>
+			</div>
```

Comment:
- [ ] `required`

The list of icons uses a `ul` tag to group them, and `li per icon;` each icon has a link (a tag).
- Apart from being semantically correct, the a tag will let you connect your social networks with the icons, using href.
- something like the screenshot below
![image](https://user-images.githubusercontent.com/66230306/192020367-d58c649f-f052-48b3-87bd-0f65620419f0.png)

please make the changes 🥂